client/ayon_core/plugins/loader/open_file.py

```diff
@@ -13,6 +15,235 @@
 )
 
 
+WINDOWS_USER_REG_PATH = (
+    r"Software\Microsoft\Windows\CurrentVersion\Explorer\FileExts"
+    r"\{ext}\UserChoice"
+)
+
+
+class _Cache:
```

Comment:
Would it make more sense to move this cache out of this action file so that it doesn't reload on plug-in rediscovery? Because currently it would, right?

Reply:
True - I figured you did this because it was slow, but it seemed fine on my machine on Windows at least so 🤷‍♂️ 
Can you add a `# TODO` to hint that moving it out may be helpful if it's slow because it currently reloads with the action reload.

Then we can leave it as is. Thanks.